### PR TITLE
zh-translation: /docs/reference/config/analysis/ist0113/index.md

### PR DESCRIPTION
**Pull request by gauliang**
ref: https://github.com/servicemesher/istio-official-translation/issues/1665

### CODE REVIEW

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 1]
```diff
@@ -25,16 +23,13 @@ spec:
...
 {{< /text >}}
 
-The effect of this policy resource is that all services have an authentication
-policy requiring mutual TLS to be used. However, note that without a corresponding
-destination rule requiring traffic to use mutual TLS, traffic will be sent to services
-without using mutual TLS. This conflict means that traffic destined for services in
-the mesh will ultimately fail.
+该策略资源的效果是,所有服务都需要使用双向 TLS 认证策略。
+但是，请注意，没有相应的目标规则要求流量使用双向 TLS，流量将在不使用 TLS 的情况下被发送到服务。
+此冲突意味着,目的地为网格中的服务的流量将最终失败。
```
Comment (yuxiaobo96) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
此冲突意味着，目的地为网格中的服务的流量将最终失败。
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 2]
```diff
@@ -48,51 +43,32 @@ spec:
...
-Look at the output of the message. You should see something like:
+## 哪些目标规则和策略与服务有关{#which-destination-rules-and-policies-are-relevant-to-a-service}
+
+为了有效解决双向 TLS 冲突，了解目标规则和策略如何影响到达服务的流量会很有帮助。
+考虑一个在 `my-namespace` 命名空间中的 `my-service` 示例服务。
+要确定哪个策略对象应用于 `my-service`，请按以下顺序匹配以下资源：
+
+1. 在命名空间 `my-namespace` 中，定义了 `target` 为 `my-service` 的策略。
+1. 在命名空间 `my-namespace` 中，名为 `default` 且没有定义 `target` 的策略。这样的策略隐式的适用于整个命名空间。
+1. 名为 `default` 的策略。
```
Comment (yuxiaobo96) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
1. 名为 `default` 的网格策略资源。
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 3]
```diff
@@ -48,51 +43,32 @@ spec:
...
+
+要确定哪些目标规则被应用于发送到 `my-service` 的流量，我们首先必须知道流量来自哪个命名空间。
+本例中，我们假设这个命名空间为 `other-namespace`。
+目标规则按照下面的顺序进行匹配：
+
+1. 命名空间 `other-namespace` 中，要匹配主机 `my-service.my-namespace.svc.cluster.local` 的目标规则，
+   这可能是完全匹配或通配符匹配。还要注意，控制配置资源可见性的 `exportTo` 字段将被忽略，因为与源服务相同的命名空间中的资源始终可见。
+1. 命名空间 `my-namespace` 中，要匹配主机 `my-service.my-namespace.svc.cluster.local` 的目标规则，
+   这可能是完全匹配或通配符匹配。请注意，`exportTo` 字段必须定义此资源为公共资源（例如，它取值为`*`或未指定），以便进行匹配。
```
Comment (yuxiaobo96) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
   这可能是完全匹配或通配符匹配。请注意，`exportTo` 字段必须定义此资源为公共资源（例如，它取值为`"*"`或未指定），以便进行匹配。
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 4]
```diff
@@ -48,51 +43,32 @@ spec:
...
+1. 根命名空间（默认为 `istio-system`）中要匹配主机 `my-service.my-namespace.svc.cluster.local` 的目标规则。
+   根命名空间由 [`MeshConfig` 资源](/zh/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig) 的 `rootNamespace` 属性控制。请注意，`exportTo` 字段必须定义此资源为公共资源（例如，它取值为`*`或未指定），以便进行匹配。
```
Comment (yuxiaobo96) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
   根命名空间由 [`MeshConfig` 资源](/zh/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig) 的 `rootNamespace` 属性控制。请注意，`exportTo` 字段必须定义此资源为公共资源（例如，它取值为`"*"`或未指定），以便进行匹配。
```

**Comment by yuxiaobo96** [comment]
@ilylia 
如果有空，请对此 pr 进行二次 review

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 5]
```diff
@@ -25,16 +23,13 @@ spec:
   - mtls: {}
 {{< /text >}}
 
-The effect of this policy resource is that all services have an authentication
-policy requiring mutual TLS to be used. However, note that without a corresponding
-destination rule requiring traffic to use mutual TLS, traffic will be sent to services
-without using mutual TLS. This conflict means that traffic destined for services in
-the mesh will ultimately fail.
+该策略资源的效果是,所有服务都需要使用双向 TLS 认证策略。
+但是，请注意，没有相应的目标规则要求流量使用双向 TLS，流量将在不使用 TLS 的情况下被发送到服务。
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
但是，请注意，如果没有相应的目标规则要求流量使用双向 TLS，流量将在不使用双向 TLS 的情况下被发送到服务。
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 6]
```diff
@@ -25,16 +23,13 @@ spec:
...
+此冲突意味着,目的地为网格中的服务的流量将最终失败。
 
-In this example, you can fix the issue in one of two ways: you could downgrade
-the mesh policy's mutual TLS requirements to accept plaintext traffic (which might include
-removing the mesh policy entirely), or you can create a corresponding
-destination rule that specifies to use mutual TLS for traffic within the mesh:
+此示例中，您可以通过以下两种方式之一解决此问题：
+您可以将网格策略的双向 TLS 要求降级，以接收明文流量（这可能包括完全删除网格策略），
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
您可以降低网格策略的双向 TLS 要求以接收明文流量（可能需要完全删除网格策略），
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 7]
```diff
@@ -48,51 +43,32 @@ spec:
...
-is used.
-
-## How to resolve
-
-Look at the output of the message. You should see something like:
+## 哪些目标规则和策略与服务有关{#which-destination-rules-and-policies-are-relevant-to-a-service}
+
+为了有效解决双向 TLS 冲突，了解目标规则和策略如何影响到达服务的流量会很有帮助。
+考虑一个在 `my-namespace` 命名空间中的 `my-service` 示例服务。
+要确定哪个策略对象应用于 `my-service`，请按以下顺序匹配以下资源：
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
要确定哪个策略对象应用于 `my-service`，以下资源将按序匹配：
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 8]
```diff
@@ -48,51 +43,32 @@ spec:
...
-Finally, be aware that Istio doesn't apply any concept of inheritance when
-following these rules. The first resources that matches the specified criteria
-is used.
-
-## How to resolve
-
-Look at the output of the message. You should see something like:
+## 哪些目标规则和策略与服务有关{#which-destination-rules-and-policies-are-relevant-to-a-service}
+
+为了有效解决双向 TLS 冲突，了解目标规则和策略如何影响到达服务的流量会很有帮助。
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
为了有效解决双向 TLS 冲突，加深对目标规则和策略是如何影响到达服务的流量的理解会很有帮助。
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 9]
```diff
@@ -48,51 +43,32 @@ spec:
...
+考虑一个在 `my-namespace` 命名空间中的 `my-service` 示例服务。
+要确定哪个策略对象应用于 `my-service`，请按以下顺序匹配以下资源：
+
+1. 在命名空间 `my-namespace` 中，定义了 `target` 为 `my-service` 的策略。
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
1. 在命名空间 `my-namespace` 中，指定 `target` 为 `my-service` 的策略。
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 10]
```diff
@@ -48,51 +43,32 @@ spec:
...
+1. 在命名空间 `my-namespace` 中，名为 `default` 且没有定义 `target` 的策略。这样的策略隐式的适用于整个命名空间。
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
1. 在命名空间 `my-namespace` 中，名为 `default` 且没有定义 `target` 的策略。这样的策略隐式的应用于整个命名空间。
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 11]
```diff
@@ -48,51 +43,32 @@ spec:
...
+1. 名为 `default` 的策略。
+
+要确定哪些目标规则被应用于发送到 `my-service` 的流量，我们首先必须知道流量来自哪个命名空间。
+本例中，我们假设这个命名空间为 `other-namespace`。
+目标规则按照下面的顺序进行匹配：
+
+1. 命名空间 `other-namespace` 中，要匹配主机 `my-service.my-namespace.svc.cluster.local` 的目标规则，
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
1. 命名空间 `other-namespace` 中，匹配主机 `my-service.my-namespace.svc.cluster.local` 的目标规则，
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 12]
```diff
@@ -48,51 +43,32 @@ spec:
...
+   这可能是完全匹配或通配符匹配。还要注意，控制配置资源可见性的 `exportTo` 字段将被忽略，因为与源服务相同的命名空间中的资源始终可见。
+1. 命名空间 `my-namespace` 中，要匹配主机 `my-service.my-namespace.svc.cluster.local` 的目标规则，
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
1. 命名空间 `my-namespace` 中，匹配主机 `my-service.my-namespace.svc.cluster.local` 的目标规则，
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 13]
```diff
@@ -48,51 +43,32 @@ spec:
...
+   这可能是完全匹配或通配符匹配。请注意，`exportTo` 字段必须定义此资源为公共资源（例如，它取值为`*`或未指定），以便进行匹配。
+1. 根命名空间（默认为 `istio-system`）中要匹配主机 `my-service.my-namespace.svc.cluster.local` 的目标规则。
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
1. 根命名空间（默认为 `istio-system`）中匹配主机 `my-service.my-namespace.svc.cluster.local` 的目标规则。
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 14]
```diff
@@ -102,27 +78,17 @@ myhost.my-namespace.svc.cluster.local:8080. The DestinationRule
...
+* 策略资源 `my-namespace/my-policy`，它指定 `Plaintext` 作为其支持的双向 TLS 模式。
+* 目标规则 `istio-system/default-rule`，它要求到达 `myhost.my-namespace.svc.cluster.local:8080` 主机的流量使用双向 TLS。
+
+您可以通过执行以下任一操作来解决冲突：
+
+* 修改策略资源 `my-namespace/my-policy` 以将双向 TLS 作为身份验证模式。
+  通常，可以通过在资源中添加一个 `peer` 属性来实现，其子属性为 `mtls`。您可以在
+  [策略对象参考](/zh/docs/reference/config/security/istio.authentication.v1alpha1/#Policy) 中阅读有关策略对象的更多信息。
+* 修改目标规则 `istio-system/default-rule`，删除 `ISTIO_MUTUAL` 以不使用双向 TLS。
+  请注意，默认情况下 `default-rule` 在 `istio-system` 命名空间中，`istio-system` 命名空间被认为是配置的根命名空间（尽管可以通过资源中的 `rootNamespace` 属性来覆盖它）。这意味着此目标规则可能会影响网格中的所有其他服务。
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
  请注意，`default-rule` 在 `istio-system` 命名空间中，默认情况下 `istio-system` 命名空间被认为是配置的根命名空间（尽管可以通过资源中的 `rootNamespace` 属性来覆盖它）。
  这意味着此目标规则可能会影响网格中的所有其他服务。
```

File: content/en/docs/reference/config/analysis/ist0113/index.md [hunk 15]
```diff
@@ -102,27 +78,17 @@ myhost.my-namespace.svc.cluster.local:8080. The DestinationRule
...
+* 在与服务所在命名空间相同的命名空间中，添加新的目标规则（本例中为 `my-namespace`），并且不要将流量策略定义为 `ISTIO_MUTUAL`。
```
Comment (ilylia) [inline content/en/docs/reference/config/analysis/ist0113/index.md]:
```suggestion
* 在与服务所在同一命名空间（本例中为 `my-namespace`）中，添加新的目标规则，并且不要将流量策略定义为 `ISTIO_MUTUAL`。
```

**Pull request closed without merging**